swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -113,8 +113,15 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
       completionHandler(error)
 
     } catch {
+      if let error = error as? AdapterError,
+         case AdapterError.connlibConnectError(let string) = error,
+         string.contains("nodename nor servname provided") {
```

Comment:
These messages are often localised by the OS so I don't think this will do much. We receive dozens of Sentry alerts with the same error from Windows machines in different languages.

If we really want to catch this, we should make the error typed and perhaps display an alert?

Reply:
Hmm, good catch. I'll investigate how easy it'd be to surface errors all the way back to the UI proc. If it's straightforward, we could simply show these with an `NSAlert()` (macOS only)